### PR TITLE
Add system dependencies to build instructions

### LINKED ISSUE #56: Error on build: "could not find system library 'fontconfig' required by the 'servo-fontconfig-sys' crate"
I get an error running following the instructions in the readme, installing the missing dependency with `sudo apt install libfontconfig-dev` fixes the error. I'll put in a PR to add that to the readme, but adding here to document it and maybe you know of a way to avoid the extra step. Full error is below:

```
error: failed to run custom build command for `servo-fontconfig-sys v5.1.0`

Caused by:
  process didn't exit successfully: `/home/henry/workspace/blackjack/target/debug/build/servo-fontconfig-sys-2618e6ca9957a126/build-script-build` (exit status: 101)
  --- stdout
  cargo:rerun-if-env-changed=FONTCONFIG_NO_PKG_CONFIG
  cargo:rerun-if-env-changed=PKG_CONFIG_x86_64-unknown-linux-gnu
  cargo:rerun-if-env-changed=PKG_CONFIG_x86_64_unknown_linux_gnu
  cargo:rerun-if-env-changed=HOST_PKG_CONFIG
  cargo:rerun-if-env-changed=PKG_CONFIG
  cargo:rerun-if-env-changed=FONTCONFIG_STATIC
  cargo:rerun-if-env-changed=FONTCONFIG_DYNAMIC
  cargo:rerun-if-env-changed=PKG_CONFIG_ALL_STATIC
  cargo:rerun-if-env-changed=PKG_CONFIG_ALL_DYNAMIC
  cargo:rerun-if-env-changed=PKG_CONFIG_PATH_x86_64-unknown-linux-gnu
  cargo:rerun-if-env-changed=PKG_CONFIG_PATH_x86_64_unknown_linux_gnu
  cargo:rerun-if-env-changed=HOST_PKG_CONFIG_PATH
  cargo:rerun-if-env-changed=PKG_CONFIG_PATH
  cargo:rerun-if-env-changed=PKG_CONFIG_LIBDIR_x86_64-unknown-linux-gnu
  cargo:rerun-if-env-changed=PKG_CONFIG_LIBDIR_x86_64_unknown_linux_gnu
  cargo:rerun-if-env-changed=HOST_PKG_CONFIG_LIBDIR
  cargo:rerun-if-env-changed=PKG_CONFIG_LIBDIR
  cargo:rerun-if-env-changed=PKG_CONFIG_SYSROOT_DIR_x86_64-unknown-linux-gnu
  cargo:rerun-if-env-changed=PKG_CONFIG_SYSROOT_DIR_x86_64_unknown_linux_gnu
  cargo:rerun-if-env-changed=HOST_PKG_CONFIG_SYSROOT_DIR
  cargo:rerun-if-env-changed=PKG_CONFIG_SYSROOT_DIR

  --- stderr
  thread 'main' panicked at '`"pkg-config" "--libs" "--cflags" "fontconfig" "fontconfig >= 2.11.1"` did not exit successfully: exit status: 1
  error: could not find system library 'fontconfig' required by the 'servo-fontconfig-sys' crate

  --- stderr
  Package fontconfig was not found in the pkg-config search path.
  Perhaps you should add the directory containing `fontconfig.pc'
  to the PKG_CONFIG_PATH environment variable
  No package 'fontconfig' found
  Package fontconfig was not found in the pkg-config search path.
  Perhaps you should add the directory containing `fontconfig.pc'
  to the PKG_CONFIG_PATH environment variable
  No package 'fontconfig' found
  ', /home/henry/.cargo/registry/src/github.com-1ecc6299db9ec823/servo-fontconfig-sys-5.1.0/build.rs:34:17
```

### PR DESCRIPTION
**Pull request by HenryWConklin**
Resolves #56

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,7 +45,10 @@ git lfs pull
 
 [^1]: Linux users can install `git-lfs` with their distro's package manager (`apt install git-lfs` / `yum install git-lfs` / `pacman -S git-lfs`). MacOS users using homebrew can use `brew install git-lfs`. Other users should follow the [git-lfs install instructions](https://git-lfs.github.com/).
 
-2. From the same folder, run `cargo run --bin blackjack_ui` to launch Blackjack.
+2. Install build dependencies:
+   * Ubuntu/Debian: `sudo apt install libfontconfig-dev`
```
Comment (setzer22) [inline README.md:49]: Could you please add a couple more popular distros? The package seems to be called `fontconfig` on Arch, and `fontconfig-devel` on Fedora.

Also a note stating that these may not be all the missing system dependencies, and asking people to file in a PR if they find any other one.

File: README.md [hunk 2]
```diff
@@ -45,7 +45,10 @@ git lfs pull
...
+
+3. From the same folder, run `cargo run --bin blackjack_ui` to launch Blackjack.
```
Comment (setzer22) [inline README.md:51]: While we're at it, could you please change the run command to `cargo run --release --bin blackjack_ui`. Users should run this on release mode (even though debug mode works fine for many use cases).

**Review by setzer22 — CHANGES_REQUESTED** [review]
Thanks for sending the PR :+1:  A couple minor things:

File: README.md (hunk 1 above)
Comment (HenryWConklin) [reply README.md]: Added, hopefully I got those commands right I'm not too familiar with other distros.

File: README.md (hunk 2 above)
Comment (HenryWConklin) [reply README.md]: Added.

**Review by setzer22 — APPROVED** [review]
Awesome, thanks again for the PR :)  will merge once CI passes

File: README.md [hunk 3]
```diff
@@ -45,7 +45,12 @@ git lfs pull
 
 [^1]: Linux users can install `git-lfs` with their distro's package manager (`apt install git-lfs` / `yum install git-lfs` / `pacman -S git-lfs`). MacOS users using homebrew can use `brew install git-lfs`. Other users should follow the [git-lfs install instructions](https://git-lfs.github.com/).
 
-2. From the same folder, run `cargo run --bin blackjack_ui` to launch Blackjack.
+2. Install build dependencies. This may not cover everything, please file an issue or a pull request if you find anything missing:
+   * Ubuntu/Debian: `sudo apt install libfontconfig-dev`
+   * Arch Linux: `sudo pacman -S fontconfig`
+   * Fedora: `sudo dnf install fontconfig-devel`
+
```
Comment (setzer22) [inline README.md:52]: Let's add a reference to the issue here:
```suggestion
> **Note**: The `fontconfig` native dependency is temporary, and will no longer be necessary once this upstream issue is fixed: https://github.com/rust-windowing/winit/issues/2373
```

**Review by setzer22 — APPROVED** [review]

**Pull request merged**